docs/guides/blocks/sync-tables/dynamic.md

```diff
@@ -252,7 +252,9 @@ For example, if you define a property with the name `GitHub (Beta)`, it will be
 Similar to property names, you must also determine the schema for each property. In many cases this involves a large `switch` statement that translates from the type descriptors in the dataset to the equivalent schemas in Coda. If a given field type doesn't have an equivalent schema in Coda it usually makes sense to fallback to a string.
 
 ```ts
-function getPropertySchema(customField): coda.Schema & coda.ObjectSchemaProperty {
+function getPropertySchema(customField)
+  :coda.Schema & coda.ObjectSchemaProperty 
```

Comment:
This doesn't look quite right/desirable, is this formatting intended for long function signature lines?

Reply:
I never know the right way to wrap long functions in TypeScript. How's this?